### PR TITLE
Feat/dockerize frontend

### PR DESCRIPTION
**Pull request by Oferlis**
Should I also add a docker-compose file?

### CODE REVIEW

File: docker-compose.yml
Comment (idodav) [inline docker-compose.yml:1]: Please remove the file

File: traffic-dashboard/Dockerfile
Comment (idodav) [inline traffic-dashboard/Dockerfile:1]: Please don't assume that the dist app exists.
Copy the entire source the dockerfile. Build the projects inside the dockerfile.
Create a multistage dockerfile. First is the builder. And the second is the final image. Copy the assets from the builder to the final base image.

**Review by idodav — CHANGES_REQUESTED** [review]

**Review by idodav — DISMISSED** [review]

**Review by SID262000 — COMMENTED** [review]
@Oferlis Please add documentation around your changes. It's extremely important to have a well documented codebase that would eventually help to onboard new developers with ease.
Also, can you please confirm in case these changes have been run through any linters, Thanks 
cc: @idodav

**Comment by idodav** [comment]
@SID262000 How do you suggest that we document such change?

**Comment by SID262000** [comment]
@idodav Please refer to this section. This will really help to improve the quality of docker files(both existing & new ones).

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

cc: @Oferlis

**Comment by idodav** [comment]
@Oferlis Is this ready for review?

**Comment by Oferlis** [comment]
@idodav Please take a look at the changes, and proxying works, but ports must be exposed.
A solution is exposing a range of ports and notifying users that these ports must be available for use by Sharkio.
What do you think? is there a better solution?
I could also open this issue to discussion in the Discord channel

**Comment by idodav** [comment]
> 

It's fine. The EXPOSE keyword is mainly for documentation.
It doesn't actually expose the ports.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: "3"
+services:
+  backend:
+    build: ./traffic-sniffer
+    ports:
```
[the remark this answers is not in the document]
Comment (idodav) [reply docker-compose.yml:5]: You could expose 5550-5560 for now.
we will figure out a different splution later.

[the remark this answers is not in the document]
Comment (Oferlis) [reply docker-compose.yml:5]: Done, PR is ready for review

**Pull request merged**